src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -666,10 +666,12 @@ public ListenableFuture<Iterable<Digest>> findMissingBlobs(
 
     if (configs.getServer().isFindMissingBlobsViaBackplane()) {
       try {
-        Map<Digest, Set<String>> foundBlobs = backplane.getBlobDigestsWorkers(blobDigests);
+        Set<Digest> distinctDigest = new HashSet<>();
```

Comment:
change this to `uniqueDigests` - collection as plural for consistency

Reply:
done